src/test/java/fshop/db/FoodDaoImplTest.java

```diff
@@ -0,0 +1,129 @@
+package fshop.db;
+
+import static junit.framework.TestCase.assertEquals;
+
+import fshop.model.Food;
+import fshop.service.workfile.ReadCsv;
+import fshop.service.workfile.ReadCsvImpl;
+import java.util.List;
+import java.util.Map;
+import org.junit.Test;
+
+public class FoodDaoImplTest {
+    @Test(expected = NullPointerException.class)
+    public void getValue_null() {
+        FoodDao foodDao = new FoodDaoImpl();
+        foodDao.addAll(new ReadCsvImpl().read("file.csv"));
+        foodDao.get(null);
+    }
+
+    @Test
+    public void getValue_test1_ok() {
```

Comment:
don't use numbers in method naming, describe the state of state or input data in some way